x-pack/plugins/endpoint/server/routes/resolver/ancestry.ts

```diff
@@ -6,13 +6,11 @@
 
 import { schema } from '@kbn/config-schema';
 import { RequestHandler, Logger } from 'kibana/server';
-import { getPaginationParams } from './utils/pagination';
-import { RelatedEventsQuery } from './queries/related_events';
+import { Fetcher } from './utils/fetch';
 import { IndexPatternRetriever } from '../../index_pattern';
 
-interface RelatedEventsQueryParams {
-  after?: string;
-  limit: number;
+interface AncestryQueryParams {
+  ancestors: number;
```

Comment:
❔ A comment explaining what this means in the context of an "Ancestor Query": Is it some upper limit to the number of ancestors that can/will be fetched?

Reply:
That's correct, it's the upper limit to the number of ancestors that will be fetched.